crowbar_framework/lib/crowbar/lock/local_blocking.rb

```diff
@@ -54,5 +51,15 @@ def release
       @locked = false
       self
     end
+
+    private
+
+    def ensure_lock_file_exists
+      @file ||= File.new(path, File::RDWR | File::CREAT, 0o644)
+    rescue
+      logger.error("Couldn't open #{path} for locking: #$!")
+      logger.error("cwd was #{Dir.getwd})")
+      raise "Couldn't open #{path} for locking: #$!"
```

Comment:
This was only refactoring this code out. I'm not sure what the possible user of this method expects on failure, so I think it's better to keep it the way it was